config/versions.json

```diff
@@ -1,5 +1,5 @@
 {
     "github.com/golang-fips/go": "main",
-    "github.com/golang-fips/openssl-fips": "b175be2ccd46683a51cba60a9a2087b09593317d",
+    "github.com/golang-fips/openssl-fips": "7544e9a0711b109a2c114b021d079263a3d683a1",
```

Comment:
As you probably noticed already, the key should be changed to "github.com/golang-fips/openssl" as the other script expects.